git-cms-init

```diff
@@ -18,25 +18,33 @@ usage () {
   $ECHO "    --check        \trun additional checks on the status of the user repository"
   $ECHO "    --https        \tuse https, rather than ssh to access your personal repository"
   $ECHO "    --ssh          \tuse ssh, rather than https to access the official repository"
+  $ECHO "    --upstream-only\tconfigure only the official upstream repository, not the user repository"
   $ECHO "-q, --quiet, -z    \tdo not print out progress"
   $ECHO "-y, --yes          \tassume yes to all questions"
   exit $1
 }
 
 # configuration
 CHECK=
-CUSTOM_REMOTE=
+CUSTOM_REMOTE=false
 PROTOCOL=mixed
-DEBUG=0
-VERBOSE=1
+DEBUG=false
+VERBOSE=true
+UPSTREAM_ONLY=false
 OAUTH_TOKEN=869cfb0477e0cae72fb233be5e1f02bd97905bad
 PROXY=`git config https.proxy` && PROXY="-x $PROXY"
 
 # colors and formatting
 RED='\033[31m'
 NORMAL='\033[0m'
 
-verbose () {
+debug() {
+  if $DEBUG; then
+    $ECHO "$@"
+  fi
+}
+
+verbose() {
   if [ "$VERBOSE" = 1 ]; then
```

Comment:
this should be `if $VERBOSE; then` ...right?

Reply:
right, going to fix it